Frameworks/Foundation/NSURLSessionConfiguration.mm

```diff
@@ -38,7 +38,7 @@ - (instancetype)init {
         _networkServiceType = NSURLNetworkServiceTypeDefault;
         _requestCachePolicy = NSURLRequestUseProtocolCachePolicy;
         _HTTPMaximumConnectionsPerHost = 6;
-        _TLSMaximumSupportedProtocol = kTLSProtocol12;
+        _TLSMaximumSupportedProtocol = kSSLProtocolUnknown;
```

Comment:
This worries me, it is not a TLS constant. Please check whether they introduced a new one for the value you are seeing.